openpype/hosts/fusion/plugins/publish/collect_instances.py

```diff
@@ -24,23 +22,62 @@ def process(self, instance):
         creator_attributes = instance.data["creator_attributes"]
         instance.data.update(creator_attributes)
 
+        frame_range_source = creator_attributes.get("frame_range_source")
+        instance.data["frame_range_source"] = frame_range_source
+
+        if frame_range_source == "asset_db":
+            # get asset frame ranges
+            start = context.data["frameStart"]
+            end = context.data["frameEnd"]
+            handle_start = context.data["handleStart"]
+            handle_end = context.data["handleEnd"]
+            start_with_handle = start - handle_start
+            end_with_handle = end + handle_end
+
+        if frame_range_source == "render_range":
+            # set comp render frame ranges
+            start = context.data["renderFrameStart"]
+            end = context.data["renderFrameEnd"]
+            handle_start = 0
+            handle_end = 0
+            start_with_handle = start
+            end_with_handle = end
+
+        if frame_range_source == "comp_range":
+            comp_start = context.data["compFrameStart"]
+            comp_end = context.data["compFrameEnd"]
+            render_start = context.data["renderFrameStart"]
+            render_end = context.data["renderFrameEnd"]
+            # set comp frame ranges
+            start = render_start
+            end = render_end
+            handle_start = render_start - comp_start
+            handle_end = comp_end - render_end
+            start_with_handle = comp_start
+            end_with_handle = comp_end
```

Comment:
This would need to be if..else and have an `else` entry raising a value error or alike for IDEs not to crazy on start, end, handle_start and handle_end possibly being undefined.

Now it could technically enter none of these if statements and leave `start` undefined, etc.

Also, with [this recent PR](https://github.com/ynput/OpenPype/pull/4978) fixing some naming elsewhere let's stick to the same naming conventions. Which would mean:
- `start` would be `frame_start`
- `end` would be `frame_end`
- `start_with_handle` would be `frame_start_handle`
- `end_with_handle` would be `frame_end_handle`

I'm not saying it's a perfect improvement and so much clearer, but at least it's consistent across the codebase and also matches the data being set as well.